examples/StandardFirmataEthernet/ethernetConfig.h

```diff
@@ -3,8 +3,9 @@
  *
  * You must configure your particular hardware. Follow the steps below.
  *
- * Currently StandardFirmataEthernet is configured as a TCP client. An
- * option to configure as a server may be added in the future.
+ * Currently StandardFirmataEthernet is configured as a TCP client. 
```

Comment:
Please change this text to:
"By default, StandardFirmataEthernet is configured as a TCP client. To configure as a TCP server, see STEP 2."

Then update STEP 2 to read as follows:
```c++
// STEP 2 [REQUIRED for all boards and shields]
// TCP Client configuration:
// To configure your board as a TCP client, set the IP address of the server you want to connect to.
// TCP Server configuration:
// To configure your board as a TCP server, comment out the following line and also ensure that
// remote_host is also commented out.
```